hs/ImageConversion.hs

```diff
@@ -1,9 +1,29 @@
 module ImageConversion
-    (createImageFile, removeImage) where
+    (createImageFile, removeImage, compileTexToPdf) where
 
 import System.Process
 import GHC.IO.Handle.Types
 
+-- | Compiles a TeX file to a PDF file.
+compileTexToPdf :: String -> IO
+                      (Maybe Handle,
+                       Maybe Handle,
+                       Maybe Handle,
+                       ProcessHandle)
+
+compileTexToPdf inName = createProcess $ CreateProcess
```

Comment:
Perhaps the filename should be changed to something that accounts for non-image-converting functions.